app/models.py

```diff
@@ -388,8 +388,8 @@ def get_service_counts(supplier_id):
         }
 
     @staticmethod
-    def serialize_agreed_variation(agreed_variation):
-        if not agreed_variation.get("agreedUserId"):
+    def serialize_agreed_variation(agreed_variation, with_users=True):
```

Comment:
`with_users=False` by default. The reason we add flags for these things is that these operations are expensive for a list of objects, so they must require and explicit opt-in. Otherwise it's easy to trigger them accidentally.